src/ontology/mondo-edit.obo

```diff
@@ -421624,6 +421624,7 @@ synonym: "lymphoid leukosis" EXACT [https://www.merckvetmanual.com/]
 xref: MESH:D001353 {source="MONDO:equivalentTo"}
 xref: UMLS:C0004421 {source="MONDO:equivalentTo"}
 is_a: MONDO:0024905 {source="MESH:D001353"} ! bird disease
+is_a: MONDO:0100321 ! viral disease or post-viral disorder
```

Comment:
the source for this subclassOf assertion is missing here

Reply:
done